app/templates/components/forms/profile-page.hbs

```diff
@@ -0,0 +1,34 @@
+<form class="ui large form" autocomplete="off" {{action 'submit' on='submit'}}>
+  <div class="ui stackable grid">
+    <div class="four wide column">
+      {{widgets/forms/image-upload
+        imageUrl=data.event.backgroundUrl
```

Comment:
Change the variable into `model.avatarUrl`. (That is how it is gonna be once we have the API call in place)